consensus/vbft/node_utils.go

```diff
@@ -235,29 +235,13 @@ func calcParticipantPeers(cfg *BlockParticipantConfig, chain *vconfig.ChainConfi
 
 	peers := make([]uint32, 0)
 	peerMap := make(map[uint32]bool)
-	proposerMap := make(map[uint32]bool)
 	var cnt uint32
 
-	if checkCalcEndorserOrCommitter(end) {
```

Comment:
I think this checking is OK.
The lowest priority proposer should be able to join endorsing/committing.

Reply:
ok